handlebars/handlebars.iml

```diff
@@ -1,20 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
hmm, I guess that would cause problems because the file would be overwritten by the Gradle importer all the time and then show changed files in git...